monero-outreach-docs/translations/de/merchants/how_to_accept_monero_de.md

```diff
@@ -0,0 +1,86 @@
+# Wie man Monero akzeptiert
+
+Wenn Du in imstande bist, eine App zu installieren, dann besitzt du die notwendigen Fähigkeiten, um Monero (XMR) in Deinem Geschäft akzeptieren zu können. Dieser Leitfaden erklärt die verbreitetsten Möglichkeiten, wie Händler Monero annehmen, und wie Du damit beginnen und darauf aufbauen kannst.
+
+Zunächst gibt es keine spezielle Art von Monero-Händlerkonto - Du benötigst lediglich eine normale Wallet und eine Adresse, um zu starten. Es wird empfohlen, eine separate Adresse für geschäftliche Transaktionen einzurichten, aus den gleichen Gründen, wie es empfohlen wird, getrennte Geschäfts- und Privatkonten zu haben. Du kannst die gleiche Wallet-Software verwenden, aber eine eigene Adresse für das Geschäft hilft, die Konten zu organisieren.
+
+Wenn Du nicht bereits eine Monero-Wallet/Adresse hast, solltest Du diese zuerst erstellen. Die offiziellen Monero GUI/CLI-Wallets von [getmonero.org/downloads](https://www.getmonero.org/downloads/) werden immer empfohlen. Außerdem wird in dieser Anleitung auf mobile Wallets verwiesen und Du kannst weitere Informationen darüber auf unserer Seite [Einrichtung der Monero-Wallet](https://www.monerooutreach.org/stories/monero-wallet-quickstart.html) finden.
+
+### _Die Anatomie einer Monero-Transaktion_
+
+Monero-Transaktionen durchlaufen die nachfolgenden Schritte. Es ist nicht notwendig, die technischen Aspekte davon vollständig zu verstehen, aber als Händler solltest Du ein Grundverständnis über diese Schritte haben, um darüber entscheiden zu können, welche Kompromisse für Dich richtig ist.
+
+1. **Einkauf** - Nachdem Du die Kosten für den Einkauf Deines Kunden in lokaler Fiatwährung errechnet und ggf. die Umsatzsteuer hinzugefügt hast, muss der entsprechende Betrag in XMR berechnet werden.
+2. **Zahlung** - Der Kunde scannt Deinen QR-Code, um Dich in XMR zu bezahlen. Die Monero-Transaktionsgebühr wird vom Kunden bezahlt und zum Zeitpunkt der Transaktion berechnet.
+3. **Transaktionspool** - Innerhalb einer Sekunde wird die Transaktion dem Monero-Netzwerk mitgeteilt und in den Transaktionspool, auch Mempool genannt, gelegt. Du kannst die Existenz Deiner Transaktion im Mempool überprüfen z. B. unter xmrchain.net/txpool.
```

Comment:
I'll add a punctuation sign behind "Sekunde". "Innerhalb einer Sekunde, wird [...]"

Reply:
That would be wrong at this point.